Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -2497,8 +2497,8 @@ unsigned RenderGrid::numTracks(GridTrackSizingDirection direction) const
 void RenderGrid::paintChildren(PaintInfo& paintInfo, const LayoutPoint& paintOffset, PaintInfo& forChild, bool usePrintRect)
 {
     ASSERT(!currentGrid().needsItemsPlacement());
-    for (RenderBox* gridItem = currentGrid().orderIterator().first(); gridItem; gridItem = currentGrid().orderIterator().next())
-        paintChild(*gridItem, paintInfo, paintOffset, forChild, usePrintRect, PaintAsInlineBlock);
+    for (RenderBox& gridItem : childrenOfType<RenderBox>(*this))
```

Comment:
Looks like we are painting the children in a different order from before. It's not super clear to me if this is ok

Reply:
definitely not ok. please do not change behavior in a cleanup PR.

Reply:
i have reverted this change